src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The truck started working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The truck stopped working ;( ");

```

Comment:
Similarly, the message in the `stopWork` method should indicate the type of machine that has stopped working. A more informative message would be: 'Truck has stopped working.'